syntax = "proto3";

// The greeting service definition.
service Category {
    // Sends a greeting
    rpc Search (CategoryRequest) returns (CategoryReply) {
    }
}

message CategoryRequest {
    string search_term = 1;
}

message CategoryReply {
    string category_name = 1;
    string category_id = 2;
    string user_id = 3;
}